templates/admin/tool_shed_repository/preview_repository.mako

```diff
@@ -0,0 +1,454 @@
+<%inherit file="/base.mako"/>
+<%namespace file="/message.mako" import="render_msg" />
+<%namespace file="/admin/tool_shed_repository/common.mako" import="*" />
+<%namespace file="/admin/tool_shed_repository/repository_actions_menu.mako" import="*" />
+
+<%def name="stylesheets()">
+    ${parent.stylesheets()}
+    ${h.css( "dynatree_skin/ui.dynatree" )}
+</%def>
+
+<%def name="javascripts()">
+    ${parent.javascripts()}
+</%def>
+${javascripts()}
```

Comment:
(Sorry, @davebx I forgot to add this part)
In order to have the script you're adding from line 19 to 339 appear in the head of the page, you can move the `</%def>` past 339. In any event, you don't need to call like 14 (it's called in base.mako IIRC) and don't need lines 11 to 13 at all if you're not adding anything to the parent's `javascripts()` call.

You can similarly move the styles defined at 339 into the stylesheets mako function on 6.